packages/bpk-component-section-header/src/BpkSectionHeader.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ReactNode } from 'react';
+
+import { cssModules } from '../../bpk-react-utils';
+import BpkText from '../../bpk-component-text';
+
+import STYLES from './BpkSectionHeader.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const SECTION_TYPES = {
+  default: 'default',
+  onDark: 'onDark',
+} as const;
+
+export type SectionType = (typeof SECTION_TYPES)[keyof typeof SECTION_TYPES];
+
+type Props = {
+  title: string;
+  description?: string;
+  button?: ReactNode;
+  type?: SectionType;
+};
+
+const BpkSectionHeader = ({
+  button,
+  description,
+  title,
+  type = SECTION_TYPES.default,
+}: Props) => {
+  const onDark = type === SECTION_TYPES.onDark;
+  return (
+    <div
```

Comment:
Could we support Accessibility in this component

Reply:
Perhaps we could add another prop as a label for the button? I don't think that the aria-label for text is needed as it's text and not an interactive element?

Reply:
button is a node, can we deal with the accessibility for it, then we don't need one more prop.

Reply:
@Supremeyh any thoughts from an accessibility perspective on what could be improved? 🙂

Reply:
Thanks @anambl and @KathyWang0208! It's great to hear that the text elements are already accessible, and interactive elements button can also support on microsite side. In this case, it's fine to me.